### DISCUSSION TITLE
Discord for informal communication

### DISCUSSION THREAD

**Discussion by kevintyj** [facebook/stylex#98] [General]
**Is your feature request related to a problem? Please describe.**
I see discussions about rollup and vite support and also conflicting contributions with docs. A synchronous informal communication channel for maintainers/contributors/users may relieve some of the issues. 

**Describe a solution you'd like**
A lot of OSS projects elect to use Discord for informal communication with the community. I would love to have a Discord server for maintainers and users of StyleX. This could help the StyleX community gain some more organization with maintainers/contributors/etc. Have seen Discord be highly effective for other OSS projects. 

**Describe alternatives you've considered**
Continue the use of Github Issues, perhaps the use of discussions may work as well.

**Additional context**
Would be happy to also have an community maintained discord server where fast communication can happen, some good examples of servers are:

- solidjs
- nx
- more examples: https://discord.com/open-source

**Comment by nmn** [comment]
Thanks for this suggestion. This is already being discussed.

**Comment by necolas** [comment]
Moving this to "discussions"

**Comment by nmn** [comment]
We will revisit this discussion in the future as the community grows.

**Comment by airtonix** [comment]
Biggest concern you'll have on this trend from those of us born  when Metallica was cool is this: 

> how do i find knowledge created in the process of these "informal" discussions?

If you look on hackernews anytime this concept pops up, mostly no-one likes discord except gamers and streamers (who care nothing for long term concerns).

I'd suggest that if you do setup a discord server: 

- encourage the use of discord threads, and
- find some kind of bot that facilitates turning a discord thread into a github discussion while notifying and updating from one to the other.

**Reply by nmn** [reply]
As I've stated, we're going to stick to using Github Discussions for now and will revisit the topic of Discord later.
